iFixit/react-commerce
# Move newsletter form into footer

This PR moves the newsletter form from the product list section to the footer, according to the new design.

## QA

1. Visit the Vercel PR preview
2. Visit parts product list page
3. Verify that the newsletter form is displayed at the bottom according to the [mockup](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit-Shopify-Working-Files?node-id=1794%3A8057)
